charts/mageai/values.yaml

```diff
@@ -42,6 +42,14 @@ service:
   # Annotations to add to the service
   annotations: {}
 
+livenessProbe:
```

Comment:
We can actually make the whole livenessProbe and readinessProbe configurable and provide the default values for the path.

Reply:
@wangxiaoyou1993 yes we could. But if we need to change one of the endpoints in the future for a different one, we could easily do this with this approach. If we use defaults, then the defaults are set on the user side and will not be overwritten. Please let me know what you think about this argument.
Im fine with both options.

Reply:
yeah. i think what you said makes sense. I still want to allow user to edit the health check path. So maybe let's make the `httpGet` overwritable. We still provide the default values in the deployment.yaml.